docs/notebooks/LORIS-API_Part3-Create_image_dataset.ipynb

```diff
@@ -0,0 +1,819 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "kernelspec": {
+      "display_name": "Python 3",
+      "language": "python",
+      "name": "python3"
+    },
+    "language_info": {
+      "codemirror_mode": {
+        "name": "ipython",
+        "version": 3
+      },
+      "file_extension": ".py",
+      "mimetype": "text/x-python",
+      "name": "python",
+      "nbconvert_exporter": "python",
+      "pygments_lexer": "ipython3",
+      "version": "3.8.3"
+    },
+    "colab": {
+      "name": "LORIS-API_Part3-Create_image_dataset.ipynb",
+      "provenance": [],
+      "collapsed_sections": [
+        "vhBrpw8LogOy",
+        "44HUGcwa7Zxu",
+        "I6NO6Mvdrt8e",
+        "MCjiEneS7Zx6",
+        "UKLJB4k77Zx8",
+        "H-u_y4QJ7ZyT",
+        "-jSSONcA7ZyY",
+        "iBC0xb937ZyG",
+        "Q7QltcmV7ZyI",
+        "EFcnVs-e7ZyN"
+      ],
+      "include_colab_link": true
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "53B8lztu7ZxQ"
+      },
+      "source": [
+        "# Image analysis with fMRI 3D images imported with Loris API"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "byKR2jTxmurK"
+      },
+      "source": [
+        "This is a tutorial to show how to use Loris' API to download fMRI images. It also contains a few examples of how the data can be used to run basic data analysis. \n",
+        "\n",
+        "This tutorial is also available as a Google colab notebook so you can run it directly from your browser. To access it, click on the button below: <a href=\"https://colab.research.google.com/github/spell00/Loris/blob/2020-08-06-JupyterCreateImageDataset/docs/notebooks/LORIS-API_Part3-Create_image_dataset.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "c1umWrZ87ZxR"
+      },
+      "source": [
+        "# Uncomment and run to install the packages required to run the notebook\n",
+        "# !pip3 install tqdm\n",
+        "# !pip3 install numpy\n",
+        "# !pip3 install nibabel\n",
+        "# !pip3 install sklearn\n",
+        "# !pip3 install matplotlib\n",
+        "# !pip3 install nilearn"
```

Comment:
Why should it be installed, I don't use it and I have no error anywhere. I can import the images, transform them in numpy tensors and display the images in the notebook just fine. 

<img width="710" alt="mri_exemple" src="https://user-images.githubusercontent.com/32504294/100160272-3df22080-2e7d-11eb-92ab-cc2b4454a71c.PNG">

Reply:
weird. maybe this is a Mac thing. Unless the files you are downloading are NIfTI files and not MINC? I was not fully surprised by the name of the package since MINC is based on the hdf5 format and h5py is HDF5 for python.